client/components/deposits-overview/strings.ts

```diff
@@ -5,4 +5,11 @@ import { __ } from '@wordpress/i18n';
 
 export default {
 	heading: __( 'Deposits', 'woocommerce-payments' ),
+	next_deposits: {
+		title: __( 'Next deposits', 'woocommerce-payments' ),
```

Comment:
This text is "Next deposit" in the current design.
```suggestion
		title: __( 'Next deposit', 'woocommerce-payments' ),
```

Reply:
Fixed, thanks! :)